src/frontend/apps/impress/src/i18n/conf.ts

```diff
@@ -1,7 +1,7 @@
 export const LANGUAGES_ALLOWED: { [key: string]: string } = {
-  en: 'English',
-  fr: 'Français',
-  de: 'Deutsch',
+  'en-us': 'English',
+  'fr-fr': 'Français',
+  'de-de': 'Deutsch',
 };
 export const LANGUAGE_COOKIE_NAME = 'docs_language';
-export const BASE_LANGUAGE = 'en';
+export const BASE_LANGUAGE = Object.keys(LANGUAGES_ALLOWED)[0];
```

Comment:
Are `LANGUAGES_ALLOWED` and `BASE_LANGUAGE` still necessary since we have about the same in the conf endpoint ?

Reply:
I implemented it using the conf endpoint, 
but this had an unintended side effects.

1. All language labels come from the backend and there, they are translated.
- If the page loads in the wrong language (whatever the reason may be) finding the right language is difficult.
2. The conf endpoint has to be refetched once the language is changed to get the conf values in the right language

I really think, we should not have translation in config values.
Should i remove the translation for the languages and re-implement it via conf endpoint?

Reply:
I implemented it, and it works nicely, check it out.